types/office-js-preview/index.d.ts

```diff
@@ -11293,14 +11292,14 @@ declare namespace Office {
          *             If more than 1,000,000 characters are passed in, an `ArgumentOutOfRange` exception is thrown.
          * @param options - An object literal that contains one or more of the following properties:-
          *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
-         *        `coercionType`: If text, the current style is applied in Outlook on the web and on Windows (classic and
-         *                      {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}.
-         *                      If the field is an HTML editor, only the text data is inserted, even if the data is HTML.
-         *                      If the data is HTML and the field supports HTML (the subject doesn't), the current style is
-         *                      applied in Outlook on the web and new Outlook on Windows. The default style is applied in classic Outlook on Windows.
-         *                      If the field is a text field, an `InvalidDataFormat` error is returned.
-         *                      If `coercionType` is not set, the result depends on the field: if the field is HTML then HTML is used;
-         *                      if the field is text, then plain text is used.
+         *        `coercionType`: If text, the current style is applied in Outlook on the web, on Windows (classic and
+         *        {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}, and on Mac.
```

Comment:
Here and other instances...

```suggestion
         *        {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}), and on Mac.
```